src/oci.h

```diff
@@ -97,7 +97,7 @@
  *  - Reacts to signals.
  *  - Exits with return code of real workload process.
  */
-#define CC_OCI_SHIM 			"cc-shim"
```

Comment:
Actually that path is given by configure, it's $libexecdir (it can be configured in ./configure and the prefix affects it for instance). You should probably provide a define from the compiler flags in Makefile.am

```
cc_shim_CPPFLAGS = -DLIBEXECDIR=\"$(libexecdir)\"
```

and in the code you can use the C string concatenation

```
#define CC_OCI_SHIM        LIBEXECDIR"/cc-shim"
````